templates/ingress.yaml

```diff
@@ -1 +1,37 @@
-# TODO: Research & add Ingress controller for exposing st2web service to public net (#6)
+{{- if .Values.ingress.enabled }}
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  {{- $name := printf "%s-%s" $.Release.Name .Values.ingress.name | trunc 63 | trimSuffix "-" | quote }}
+  name: {{ $name }}
+  labels:
+    app: {{ $name }}
+    chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
+    release: "{{ $.Release.Name }}"
+    heritage: "{{ $.Release.Service }}"
+  annotations:
+    {{- if .Values.ingress.tls }}
+    ingress.kubernetes.io/secure-backends: "true"
+    {{- end }}
+    {{- range $key, $value := .annotations }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
+spec:
+  rules:
+  {{- range .Values.ingress.hosts }}
+  - host: {{ .name }}
+    http:
+      paths:
+      {{- range .paths }}
+        - path: {{ default "/*" .path }}
+          backend:
+            serviceName: {{ .serviceName }}
+            servicePort: {{ .servicePort }}
```

Comment:
serviceName/servicePort should be optional and default to templated version of st2web service we already deploy in the cluster.
This way user won't need to guess anything from the `values.yaml` perspective or even worse hardcode releaseName from values.yaml because we didn't template it properly here.